### PR TITLE
configs: BuildWithDNF5 for ELN

### PR DESCRIPTION
**Pull request by praiskup**
Relates: https://pagure.io/releng/issue/11895

### CODE REVIEW

**Comment by sgallagher** [comment]
Let's hold off on this for a few more days; I've got two related MRs out that we should land first:

1. https://github.com/minimization/content-resolver-input/pull/1105 - makes sure ELN has `dnf5` and `dnf5-plugins` explicitly included. They're present right now, but only as part of ELN Extras.
2. https://pagure.io/fedora-kickstarts/pull-request/1025 - Once the above is merged, we will make `dnf5[-plugins]` part of the bootstrap image.

Then we can avoid needing to turn off the `bootstrap_ready` option.

**Comment by yselkowitz** [comment]
Can this proceed now?

**Comment by praiskup** [comment]
Yeah, we have to...  this is breaking Copr builds.

File: mock-core-configs/etc/mock/templates/fedora-eln.tpl [hunk 1]
```diff
@@ -12,17 +12,24 @@ config_opts['chroot_setup_cmd'] = 'install bash bzip2 coreutils cpio diffutils f
...
 # Per https://github.com/fedora-eln/eln/issues/164 updated up to 4 times a day.
 # Docs: https://docs.fedoraproject.org/en-US/eln/deliverables/#_container_image
 config_opts['bootstrap_image'] = 'quay.io/fedoraci/fedora:eln'
-# Per https://github.com/fedora-eln/eln/issues/166
-config_opts['bootstrap_image_ready'] = True
+
+# Used to be True per https://github.com/fedora-eln/eln/issues/166 but with
+# the 'dnf5' we are switching to False, reobserve after
+# https://fedoraproject.org/wiki/Changes/ReplaceDnfWithDnf5 applied to ELN!
+config_opts['bootstrap_image_ready'] = False
```
Comment (praiskup) [inline mock-core-configs/etc/mock/templates/fedora-eln.tpl:27]: Unfortunately, we made this change in ELN stack in the wrong order -> the ELN image now contains `dnf5` and not `dnf4`, so it stays "ready".  Going to update..

**Comment by FrostyX** [comment]
I just tested the proposed config and it works well.
+1

**Pull request merged**